packages/language-server/src/server.ts

```diff
@@ -81,7 +81,7 @@ connection.onInitialize(async (params) => {
 				defaultConfig: core.ConfigService.merge(core.VanillaConfig, {
 					env: { gameVersion: initializationOptions?.gameVersion },
 				}),
-				cacheRoot: fileUtil.ensureEndingSlash(url.pathToFileURL(cacheRoot).toString()),
+				cacheRoot,
 				externals,
```

Comment:
If `cacheRoot` is now given when creating `externals`, is it still necessary to store this separately in `Project`?

Reply:
The fact that we pass `cacheRoot` to `externals` is more of a side effect of how `NodeJsExternals` functions -- it needs a path to store the cached responses under. It won't exist if `BrowserExternals` is used, for example, so I'd still like to keep passing it to `Project` as well.

Reply:
Oh I guess it is still used to cache the symbol table and virtual uris